src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -523,13 +523,22 @@ const NetworkModificationNodeEditor = () => {
                         if (data[0][key] === null) {
                             delete data[0][key];
                         }
+                        for (var i = 0; i < copiedModifications.length; i++) {
+                            if (modificationUuid === copiedModifications[i]) {
+                                setCopiedModifications([]);
```

Comment:
Do not forget to add a `snackInfo` output with the specified message for this feature